gensim/models/nmf.py

```diff
@@ -3,9 +3,11 @@
 Implements online non-negative matrix factorization algorithm, which allows for fast latent topic inference.
 This NMF implementation updates in a streaming fashion and works best with sparse corpora.
 
+The shape of the input corpus is ``(n_words, n_documents)``, which is the dimension order used in the original paper.
```

Comment:
Whoa, that doesn't sound right. The input corpus must be a stream of documents, like the rest of Gensim.

I thought this only affected the (barely documented) CSC code path?

Reply:
No, you got it right, it accepts streams too, I'll make it clearer.

Reply:
Document streams are the primary interface. Other input format options can be mentioned too if supported, but not with the same strength.

We don't want people passing in CSC matrices, then complaining on our forum they're running out of memory…

Reply:
Yeah, I agree, just wanted to make it clear that the internal matrices are transposed in contrast to the ordinary LDA.

I'll just mention csc in the parameters section of the methods then.